src/vma/dev/ring_simple.cpp

```diff
@@ -215,8 +215,7 @@ void ring_simple::create_resources(ring_resource_creation_info_t* p_ring_info, b
 
 #if defined(DEFINED_IBV_EXP_FLOW_TAG)
```

Comment:
There is inconsistency in checking configuration option as `#if defined(DEFINED_IBV_EXP_FLOW_TAG)` and `#ifdef DEFINED_IBV_EXP_FLOW_TAG`